x-pack/plugins/security_solution/cypress/e2e/timelines/bulk_add_to_timeline.cy.ts

```diff
@@ -33,8 +35,9 @@ const assertFirstPageEventsAddToTimeline = () => {
   });
 };
 
-const assertAllEventsAddToTimeline = () => {
-  selectAllEvents();
+const assertAllEventsAddToTimeline = (type: 'alerts' | 'events') => {
+  if (type === 'alerts') selectAllAlerts();
```

Comment:
We should not have conditionals inside the testing code.

Reply:
@MadameSheema , I remember you told me that before so did not add any utility function `tasks` with conditionals and created a local helper. If I change, I will to duplicate the code. Could you please help me understand why `conditional` here is bad? Thanks.

Reply:
@MadameSheema , it is done.